tests/test_manifest.py

```diff
@@ -798,4 +798,4 @@ def test_get_manifest_with_files(self, helpers, component, datasetId):
             assert n_rows == 4
         elif component == "BulkRNA-seqAssay":
             assert filename_in_manifest_columns
-            assert n_rows == 3
+            assert n_rows == 4
```

Comment:
Nit: Does it make sense to map to specific fileview snapshot version so that this doesn't happen? 

In the long term future, it would be helpful to think about tests with setup and teardown methods so that they are stable.

Reply:
The overall concern is that you want to know explicitly _why_ a test failed. It's not ideal to have to figure out that the test failing is due to something updated on Synapse not directly related to the code.

Reply:
Yeah I completely agree with what Tom said. I could create a separate ticket for this: https://sagebionetworks.jira.com/browse/FDS-2306, and this would be a good way to drive to cleaner and more organized code.